data/nav.yml

```
@@ -444,6 +444,9 @@
               pill: "beta"
             - name: "Teams"
               path: "apis/rest-api/teams"
+            - name: "Team Pipelines"
```

Removing title case and aligning with other usages:

```suggestion
            - name: "Team pipelines"
```